bases/br_ibge_pof/caderneta_coletiva_2017/table_config.yaml

```diff
@@ -0,0 +1,467 @@
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: id_codigo_5_bd
+      bigquery_type: string
+      description: Código de 5 dígitos do item
+      temporal_coverage:
+          - 2017-2019
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V9001
+      bigquery_type: string
+      description: Código do tipo de despesa/aquisição
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V9002
+      bigquery_type: string
+      description: Forma de aquisição
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: yes
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V9004
+      bigquery_type: string
+      description: Código do local de aquisição
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V9007
+      bigquery_type: string
+      description: Código da Unidade de Medida pela qual o produto foi adquirido,
+          aplicável apenas para alimentos e bebidas
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V9009
+      bigquery_type: string
+      description: Código do Peso/Volume referente a Unidade de Medida pela qual o
+          produto foi adquirido, aplicavel apenas para Alimentos e Bebidas
+      temporal_coverage:
+          - 2017(1)2018
+      covered_by_dictionary: no
+      directory_column:
+          dataset_id:
+          table_id:
+          column_name:
+      measurement_unit:
+      has_sensitive_data: no
+      observations:
+      is_in_staging: true
+      is_partition: false
+    - name: V9005
+      bigquery_type: float64
+      description: Quantidade do produto adquirido, aplicável apenas para alimentos
```

Comment:
por que é float64? se for quantidade, pensei que seria int64.

Reply:
é float msm :) tem umas quantidades quebradas, cm virgula